packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -104,17 +104,16 @@ export class GCFBootstrapper {
         request.get('x-github-delivery') ||
         request.get('X-GitHub-Delivery') ||
         '';
-      const delivery =
-        request.get('x-hub-signature') || request.get('X-Hub-Signature') || '';
+      // TODO: add test to validate this signature is used on the initial
```

Comment:
Can we up-level this to a GitHub issue so we don't lose it?

Reply:
done: https://github.com/googleapis/repo-automation-bots/issues/470